com.unity.render-pipelines.core/Runtime/RenderGraph/RenderGraph.cs

```diff
@@ -4,6 +4,9 @@
 using UnityEngine.Rendering;
 using UnityEngine.Rendering.RendererUtils;
 
+// Typedef for the in-engine RendererList API (to avoid conflicts with the experimental version)
+using CoreRendererListDesc = UnityEngine.Rendering.RendererUtils.RendererListDesc;
```

Comment:
`using RendererListDesc = UnityEngine.Rendering.RendererUtils.RendererListDesc;`
did not work in this case, because we are using it inside the Experimental.Rendering namespace, which also includes the other RendererListDesc definition, so I had to "rename" the class in the alias/typedef.

In other places that we are outside of the experimental namespace (see HDRenderPipeline.cs) the rename in the alias is not necessary.

If you think there is a better or more elegant way to solve the ambiguity let me know!